src/cgns_utilities.F90

```diff
@@ -2603,7 +2603,7 @@ subroutine convertPlot3d(pFile, cFile)
     integer(kind=4) :: nZones, i, j, k, cg, dims(9), zoneID, ierr, Cx, Cy, Cz, iZone
     integer(kind=4), dimension(:, :), allocatable :: sizes
     real(kind=8), dimension(:, :, :), allocatable :: coorX, coorY, coorZ
-    character*12 :: zoneName
+    character(len=12) :: zoneName
```

Comment:
Please check that this is correct

Reply:
I came to the same independent conclusion for updating the code.